x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -96,60 +95,63 @@ export const getColumns = (
       field: 'method',
       name: i18n.COLUMN_METHOD,
       truncateText: true,
+      width: '16%',
     },
     {
       field: 'severity',
       name: i18n.COLUMN_SEVERITY,
       render: (value: TableData['severity']) => <SeverityBadge value={value} />,
       truncateText: true,
+      width: '16%',
     },
     {
-      field: 'lastCompletedRun',
+      field: 'statusDate',
       name: i18n.COLUMN_LAST_COMPLETE_RUN,
-      render: (value: TableData['lastCompletedRun']) => {
+      render: (value: TableData['statusDate']) => {
         return value == null ? (
           getEmptyTagValue()
         ) : (
-          <PreferenceFormattedDate value={new Date(value)} />
+          <FormattedDate value={value} fieldName={i18n.COLUMN_LAST_COMPLETE_RUN} />
         );
       },
       sortable: true,
       truncateText: true,
-      width: '16%',
+      width: '20%',
     },
     {
-      field: 'lastResponse',
+      field: 'status',
       name: i18n.COLUMN_LAST_RESPONSE,
-      render: (value: TableData['lastResponse']) => {
-        return value == null ? (
-          getEmptyTagValue()
-        ) : (
+      render: (value: TableData['status']) => {
+        const color =
+          value == null
+            ? 'subdued'
+            : value === 'succeeded'
+            ? 'success'
+            : value === 'failed'
+            ? 'danger'
+            : value === 'executing'
+            ? 'warning'
+            : 'subdued';
+        return (
           <>
-            {value.type === 'Fail' ? (
-              <EuiTextColor color="danger">
-                {value.type} <EuiIconTip content={value.message} type="iInCircle" />
-              </EuiTextColor>
-            ) : (
-              <EuiTextColor color="secondary">{value.type}</EuiTextColor>
-            )}
+            <EuiHealth color={color}>{value ?? getEmptyTagValue()}</EuiHealth>
           </>
         );
       },
+      width: '16%',
       truncateText: true,
     },
     {
       field: 'tags',
       name: i18n.COLUMN_TAGS,
       render: (value: TableData['tags']) => (
-        <div>
-          <>
-            {value.map((tag, i) => (
-              <EuiBadge color="hollow" key={i}>
-                {tag}
-              </EuiBadge>
-            ))}
-          </>
-        </div>
+        <>
+          {value.map((tag, i) => (
+            <EuiBadge color="hollow" key={i}>
```

Comment:
tag names should be unique, that would make for a better key than the index here of `i` which is dangerous.